ingestion/src/metadata/cli/backup.py

```diff
@@ -98,13 +106,64 @@ def upload_backup(endpoint: str, bucket: str, key: str, file: Path) -> None:
         raise err
 
 
+def upload_backup_azure(account_url: str, container: str, file: Path) -> None:
+    """
+    Upload the mysqldump backup file.
+
+    :param account_url: Azure account url
+    :param container: Azure container to upload file to
+    :param file: file to upload
+    """
+
+    try:
+        from azure.identity import DefaultAzureCredential
+        from azure.storage.blob import BlobServiceClient
+
+        default_credential = DefaultAzureCredential()
+        # Create the BlobServiceClient object
+        blob_service_client = BlobServiceClient(
+            account_url, credential=default_credential
+        )
+    except ModuleNotFoundError as err:
+        logger.debug(traceback.format_exc())
+        logger.error(
+            "Trying to import DefaultAzureCredential to run the backup upload."
+        )
+        raise err
+
+    click.secho(
+        f"Uploading {file} to {account_url}/{container}...",
+        fg="bright_green",
+    )
+
+    try:
+        # Create a blob client using the local file name as the name for the blob
+        blob_client = blob_service_client.get_blob_client(
+            container=container, blob=file.name
+        )
+
+        # Upload the created file
+        with open(file=file.absolute, mode="rb", encoding=UTF_8) as data:
+            blob_client.upload_blob(data)
+
+    except ValueError as err:
+        logger.debug(traceback.format_exc())
+        logger.error("Revisit the values of --upload")
+        raise err
+    except Exception as err:
+        logger.debug(traceback.format_exc())
+        logger.error(err)
+        raise err
+
+
 def run_backup(  # pylint: disable=too-many-arguments
```

Comment:
```suggestion
def run_backup(  # pylint: disable=too-many-arguments,too-many-locals
```

Reply:
we can fix that later, as we have a ticket to restructure how we are passing the parameters